daemon/daemon_unix.go

```diff
@@ -1269,7 +1269,7 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 	// for the period and runtime as this limits what the children can be set to.
 	daemon.initCgroupsPath(filepath.Dir(path))
 
-	_, root, err := cgroups.FindCgroupMountpointAndRoot("cpu")
+	root, err := cgroups.FindCgroupMountpoint("cpu")
```

Comment:
If we don't need the _root_, the `root` variable should probably be renamed as well